app/src/main/res/layout/fragment_add_on_details.xml

```diff
@@ -21,115 +21,143 @@
             android:id="@+id/details"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="20dp"
             android:textColor="?primaryText"
             android:textColorLink="?aboutLink"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
             tools:text="@tools:sample/lorem/random" />
 
         <TextView
             android:id="@+id/author_label"
+            style="@style/AboutHeaderContentText"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_below="@+id/details"
-            style="@style/AboutHeaderContentText"
-            android:text="@string/mozac_feature_addons_authors" />
+            android:text="@string/mozac_feature_addons_authors"
+            android:layout_marginTop="20dp"
+            app:layout_constraintEnd_toStartOf="@+id/author_text"
+            app:layout_constraintHorizontal_bias="0.5"
```

Comment:
Nit: 0.5 bias is the default, and also bias won't have any effect in our case, we could remove this line from everywhere it's used in this layout